genestack_client/expression_navigator.py

```diff
@@ -2,7 +2,7 @@
 
 
 class _BaseExpressionNavigator(Application):
-    def _create_file(self, groups, organism=None, normalised_input=None, options=None):
```

Comment:
Need to have both keys, for deprecation period.

Reply:
I've returned this parameter back